groups/live/templates/bulk_gateway_user_data.tpl

```diff
@@ -75,6 +81,59 @@ mkdir -p /batenvp1rep/archive
 chmod 777 /batenvp1rep/archive
 chown batenvp1rep:batch /batenvp1rep/archive
 
+#Create /home/bulk-live/bin dir and permissions & ownership
+mkdir -p /home/bulk-live/bin
+chmod 777 /home/bulk-live/bin
+chown bulk-live:bulk-live /home/bulk-live/bin
+
+#Create /home/gateway/bin dir and permissions & ownership
+mkdir -p /home/gateway/bin
+chmod 775 /home/gateway/bin
+chown gateway:gateway /home/gateway/bin
+
+#Create /home/gateway/bin/gazette dir and permissions & ownership
+mkdir -p /home/gateway/bin/gazette
+chmod 775 /home/gateway/bin/gazette
+chown gateway:gateway /home/gateway/bin/gazette
+
+#Create /home/gateway/bin/letters dir and permissions & ownership
+mkdir -p /home/gateway/bin/letters
+chmod 775 /home/gateway/bin/letters
+chown gateway:gateway /home/gateway/bin/letters
+
+#Create /home/gateway/bin/returnedmail dir and permissions & ownership
+mkdir -p /home/gateway/bin/returnedmail
+chmod 775 /home/gateway/bin/returnedmail
+chown gateway:gateway /home/gateway/bin/returnedmail
+
+#Create /home/gateway/bin/authcode dir and permissions & ownership
+mkdir -p /home/gateway/bin/authcode
+chmod 775 /home/gateway/bin/authcode
+chown gateway:gateway /home/gateway/bin/authcode
+
+#Create /home/gateway/bin/eshu dir and permissions & ownership
+mkdir -p /home/gateway/bin/eshu
+chmod 775 /home/gateway/bin/eshu
+chown gateway:gateway /home/gateway/bin/eshu
+
+#Create /home/gateway/bin/lfp_sage_files dir and permissions & ownership
+mkdir -p /home/gateway/bin/lfp_sage_files
+chmod 775 /home/gateway/bin/lfp_sage_files
+chown gateway:gateway /home/gateway/bin/lfp_sage_files
+
+#Script that copies deployment scripts from Git repo
+touch /root/script_deploy.sh
+chmod 755 /root/script_deploy.sh
+echo '#!/bin/bash' > /root/script_deploy.sh
+echo '#Script to clone deploy scripts from Git repo and copy to various locations' >> /root/script_deploy.sh
+echo 'pushd /tmp' >> /root/script_deploy.sh
+echo 'git clone git@github.com:companieshouse/chips-service-admin.git' >> /root/script_deploy.sh
+echo 'cp -pr /tmp/chips-service-admin/scripts/bulk-outputs/gateway/* /home/bulk-live/bin/' >> /root/script_deploy.sh
+echo 'cp -pr /tmp/chips-service-admin/scripts/bulk-outputs/gazette/* /home/gateway/bin/gazette/' >> /root/script_deploy.sh
+echo 'cp -pr /tmp/chips-service-admin/scripts/bulk-outputs/letters/* /home/gateway/bin/letters/' >> /root/script_deploy.sh
+echo 'cp -pr /tmp/chips-service-admin/scripts/bulk-outputs/returnedmail/* /home/gateway/bin/returnedmail/' >> /root/script_deploy.sh
+echo 'rm -rf /tmp/chips-service-admin' >> /root/script_deploy.sh
```

Comment:
You should use the EOF option if you need to create a lot lines in a file:
```
cat <<EOF >> /root/script_deploy.sh
<script>
EOF
```

Reply:
Update as requested:

cat <<EOF >> /root/script_deploy.sh
#!/bin/bash
#Script to clone deploy scripts from GitHub repo and copy to various locations

#Clone from GitHub repo
pushd /tmp
git clone git@github.com:companieshouse/chips-service-admin.git

#Copy script to relevant paths
cp -pr /tmp/chips-service-admin/scripts/bulk-outputs/gateway/* /home/bulk-live/bin/
cp -pr /tmp/chips-service-admin/scripts/bulk-outputs/gazette/* /home/gateway/bin/gazette/
cp -pr /tmp/chips-service-admin/scripts/bulk-outputs/letters/* /home/gateway/bin/letters/
cp -pr /tmp/chips-service-admin/scripts/bulk-outputs/returnedmail/* /home/gateway/bin/returnedmail/

#Update ownership after scripts deployment
chown -R bulk-live:bulk-live /home/bulk-live/bin/
chown -R gateway:gateway /home/gateway/bin/gazette/
chown -R gateway:gateway /home/gateway/bin/letters/
chown -R gateway:gateway /home/gateway/bin/returnedmail/

#Remove cloned GitHub repo
rm -rf /tmp/chips-service-admin
EOF